roles/apicast/templates/apicast-services-grafanadashboard.yaml

```
@@ -0,0 +1,571 @@
+apiVersion: integreatly.org/v1alpha1
+kind: GrafanaDashboard
+metadata:
+  name: threescale-apicast-services
+  namespace: "{{ meta.namespace }}"
+  labels:
+    app: 3scale-api-management
+    threescale_component: apicast
+    "{{ grafana_dashboard.label.key | default(grafana_dashboard_label_key) }}": "{{ grafana_dashboard.label.value | default(grafana_dashboard_label_value) }}"
+spec:
+  name: threescale-apicast-services.json
```

Update dashboard json file name:
```suggestion
  name: "{{ meta.namespace }}-threescale-apicast-services.json"
```